website/source/layouts/docs.erb

```diff
@@ -246,6 +246,30 @@
       <li<%= sidebar_current("docs-faq") %>>
         <a href="/docs/faq.html">FAQ</a>
       </li>
+
+      <hr>
+
+      <li<%= sidebar_current("docs-consul-enterprise") %>>
```

Comment:
Same thing as I said above, I'd shorten this to docs-enterprise and update everywhere